### DISCUSSION TITLE
UBL?

### DISCUSSION THREAD

**Discussion by Cal-LF** [Jyers/Marlin#211] [Q&A] [answered]
What does UBL functionality mean? What is the "No Probe" option?

**Comment by t3chguy** [comment, marked as answer]
https://marlinfw.org/docs/features/unified_bed_leveling.html

**Comment by Schnabulation** [comment]
TL;DR: Basically it's a better version of bilinear bed leveling. You create a highly detailed mesh with 10x10 points and save this to EEPROM. Before very print you just probe 3 or 4 points and tilt the already created mesh according to these points.
